interfaces/sdcore_config/v0/interface_tests/test_requirer.py

```diff
@@ -5,33 +5,17 @@
 from scenario import Relation, State
 
 
-def test_state_active_on_joined():
+def test_no_data_on_joined():
     t = Tester(
         State(
             relations=[
                 Relation(
                     endpoint="sdcore_config",
                     interface="sdcore_config",
+                    remote_app_data={"webui_url": "some_url:123"},
                 )
             ],
         )
     )
-    state_out: State = t.run('sdcore-config-relation-joined')
-    assert state_out.unit_status.name == 'active'
-
-
-def test_state_blocked_on_broken():
-    t = Tester(
-        State(
-            relations=[
-                Relation(
-                    endpoint="sdcore_config",
-                    interface="sdcore_config",
-                )
-            ],
-        )
-    )
-    state_out: State = t.run('sdcore-config-relation-broken')
-    assert state_out.unit_status.name == 'blocked'
-
-
+    t.run('sdcore-config-relation-joined')
+    t.assert_relation_data_empty()
```

Comment:
surprised this would pass, since you are putting some data in the input state. Is the charm supposed to delete the data? If so, rename the test to something more explicit or add some comments to explain what's going on.